app/models/deploy.rb

```diff
@@ -119,7 +119,7 @@ def self.expired
   end
 
   def url
-    Rails.application.routes.url_helpers.project_deploy_path(project, self)
+    Rails.application.routes.url_helpers.project_deploy_url(project, self)
```

Comment:
hmmm yeah that url helper thing might not be avoidable for
mails/notifications ... maybe they can do `.url_for(deploy)`

On Thu, Apr 16, 2015 at 9:52 AM, Fábio Neves <notifications@github.com>
wrote:

> In app/models/deploy.rb
> <https://github.com/zendesk/samson/pull/324#discussion_r28527771>:
>
> > @@ -119,7 +119,7 @@ def self.expired
> >    end
> >
> >    def url
> > -    Rails.application.routes.url_helpers.project_deploy_path(project, self)
> > +    Rails.application.routes.url_helpers.project_deploy_url(project, self)
>
> Yeah... I also find it non natural. But I needed to display the url on the
> Flowdock notification and there was this code here displaying the path. I
> can just move my code out of the model and into the FlowdockNotification as
> it was. This will allow me to remove the line in (test.rb#L57)[
> https://github.com/zendesk/samson/blob/fneves/buddy_request_plugin/config/environments/test.rb#L57
> ]
>
> Was just trying to reuse some code that already existed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/324/files#r28527771>.
>